content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -179,8 +231,8 @@ spec:
   - address: <IPofCluster2IngressGateway>
     network: external
     ports:
-      http1: 15443 # Do not change this port value
-      tcp2: 15443 # Do not change this port value
```

Comment:
Instead of removing, maybe change to a comment that says it must be port 15433 or the corresponding node port